### ISSUE TITLE
When tour guide, can't we interact with real UI?

### ISSUE THREAD

**Issue by eugene-zheng0208** [okgrow/react-native-co-pilot#6]
I think in your version when we tour, only modal seems to be working, not real UI component working
People can interact with real UI even when tour guide mode so that they can experience really

**Comment by uppercase-army** [comment]
I agree, that would be awesome. E.g. if one of the steps wraps around a button component, if the user can then tap on the button, they complete the step and moves to the next one.

**Comment by mohebifar** [comment]
It should be possible to interact with the actual element. Do you mean navigating through the copilot steps with the interaction?

**Comment by jzabala** [comment]
@mohebifar In the issue #48 you mentioned that you had incompatibility problems between Android and iOS when you tried to implement this feature. I want to ask you if it was because of zIndex and how it works on iOS?

I tried to do this and my best solution was to set opacity=0 in the original component and re-render it on top of the modal.

**Comment by lomo009** [comment]
+1 this would so much better if you could touch what is being highlighted............

**Comment by RichardLitt** [comment]
I'm not sure that @mohebifar's question to @eugene-zheng0208 has been answered, which would help this issue along. @eugene-zheng0208: can you clarify?
